server/src/main/scala/za/co/absa/atum/server/api/controller/FlowControllerImpl.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.api.controller
+
+import za.co.absa.atum.model.dto.{CheckpointQueryDTO, CheckpointDTO}
+import za.co.absa.atum.server.api.service.FlowService
+import za.co.absa.atum.server.model.ErrorResponse
+import zio._
+
+class FlowControllerImpl(flowService: FlowService)
+  extends FlowController with BaseController {
+
+  override def getFlowCheckpoints(checkpointQueryDTO: CheckpointQueryDTO): IO[ErrorResponse, Seq[CheckpointDTO]] = {
+    serviceCall[Seq[CheckpointDTO], Seq[CheckpointDTO]](
+      flowService.getFlowCheckpoints(checkpointQueryDTO),
+      r => r
```

Comment:
I think this can be changed to `checkpointFlows` or something that describe what is being returned